bears/vcs/git/GitCommitBear.py

```diff
@@ -41,13 +50,36 @@ def get_body_checks_metadata(cls):
             omit={'self', 'body'})
 
     @classmethod
+    def get_issue_checks_metadata(cls):
+        return FunctionMetadata.from_function(
+            cls.check_issue,
+            omit={'self', 'body'})
+
+    @classmethod
     def get_metadata(cls):
         return FunctionMetadata.merge(
             FunctionMetadata.from_function(
                 cls.run,
                 omit={'self', 'dependency_results'}),
             cls.get_shortlog_checks_metadata(),
-            cls.get_body_checks_metadata())
+            cls.get_body_checks_metadata(),
+            cls.get_issue_checks_metadata())
+
+    @staticmethod
+    def get_host_from_remotes():
+        """
+        Retrieve the host from supplied git remotes.
+        """
+        remotes, _ = run_shell_command('git config --get-regex url')
+        remotes = [url.split()[-1] for url in remotes.splitlines()]
+        for url in remotes:
+            netloc = urlparse(url)[2]
+            if 'git@' in url:
+                netloc = re.findall(r'@(\S+):', url)[0]
+            else:
+                netloc = urlparse(url)[1]
+            host = netloc.split('.')[0]
+        return host
```

Comment:
fails with an exception when no remotes are set, you should set `host` to `None` before^^